_includes/nav-bar.html

```diff
@@ -34,6 +34,8 @@
             <li><a href="/documentation/faqs/">FAQS</a></li>
             <li role="separator" class="divider"></li>
             <li><a href="/code-api/">Code API</a></li>
+            <li role="separator" class="divider"></li>
+            <li><a href="/plugins/">Plugin Interfaces</a></li>
```

Comment:
And perhaps move it up a bit. I think "FAQs" and "Code API" should be the last on that list, but that's a personal preference.

Also, the links to the plugin documentation should be added to  `documentation/index.markdown` (to keep it consistent with the nav bar).